app/views/ReportView/components/PatientInformation/index.tsx

```diff
@@ -44,8 +44,11 @@ const PatientInformation = ({
     if (report?.ident) {
       const getData = async () => {
         let biopsyCollectionDate;
-        if (appendices && appendices.sampleInfo && appendices.sampleInfo[0]['Sample'] === 'Tumour') {
-          biopsyCollectionDate = appendices.sampleInfo[0]['Collection Date'];
+        if (appendices && appendices.sampleInfo) {
+          appendices.sampleInfo.map((info)=>{
```

Comment:
instead of map you can just use forEach, because map creates another dynamic array -- takes up a bit more space than searching an already existing array

Reply:
Can she use .find() too?

Reply:
`find` works fine too yeah, it short circuits which is even better

Reply:
learned!